libsolidity/ast/AST.cpp

```diff
@@ -514,20 +514,16 @@ ModifierDefinition const& ModifierDefinition::resolveVirtual(
 	solAssert(_searchStart == nullptr, "Used super in connection with modifiers.");
 
 	// If we are not doing super-lookup and the modifier is not virtual, we can stop here.
```

Comment:
The comment talks about super lookup. Why can we ignore it? Can you modify the comment and add an assertion?

Reply:
Ah, sorry, did not see the assert above. Can you modify the comment, please?